src/local-development/create-local-connection.ts

```diff
@@ -34,11 +33,12 @@ async function onCreateLocalConnectionClick(file: vscode.Uri) {
 			'Using of multiple remote origins are not fully compatible with connections',
 			{
 				modal: true,
-				detail: 'You have multiple origins defined. This is a complication for connections, ' +
-				'which have the autogenerated ID and then cannot be same on all origins. ' +
-				`If you will continue, the new connection will be designed for the first origin named "${
-					makecomappJson.origins[0].label ?? makecomappJson.origins[0].appId
-				}".`,
+				detail:
+					'You have multiple origins defined. This is a complication for connections, ' +
+					'which have the autogenerated ID and then cannot be same on all origins. ' +
+					`If you will continue, the new connection will be designed for the first origin named "${
```

Comment:
I do not fully understand the last sentence "If you will continue, the new connection will be designed for the first origin named" .. can you reformulate?

Reply:
Thanks for pointing to this. Actually, this warning is not needed anymore because of complete refactorization. Removing this part.